docs/content/doc/usage/reverse-proxies.en-us.md

```diff
@@ -169,7 +177,18 @@ In case you already have a site, and you want Gitea to share the domain name, yo
 
 ```
 git.example.com {
-    proxy /git/ http://localhost:3000 # Note: Trailing Slash after /git/
+    route /git/* {
+        uri strip_prefix /git
```

Comment:
the `/git` prefix shouldn't be stripped, although I am not familiar enough with caddy v2 to provide the exact syntax

Reply:
woops. you are right. sorry about that, I assumed incorrectly about how caddy2 operated.

Thanks for PR :) I will merge this now.